packages/pages/src/common/src/template/types.ts

```diff
@@ -142,8 +142,6 @@ export type Manifest = {
   };
   /** A map of project roots to their paths */
   projectFilepaths: {
-    /** The folder path where all template files live */
-    templatesRoot: string;
```

Comment:
I think we should leave ProjectStructure the same and just not have it be an arg to the vite plugin for now. Pretty much none of the other fields are going to work correctly if you change them either. Maybe they will? I haven't tested. But there are definitely quite a few hardcoded things. If you check the plugin's actual code I think it hardcodes the hydration folder name too.

Reply:
Ok I'll revert and remove the ability to pass in your own project structure config